### ISSUE TITLE
[BUG] QM/excavator 0.5.4.1 RC closing HWMonitor

### ISSUE THREAD

**Issue by tac-git** [nicehash/NiceHashQuickMiner#855]
**Describe the bug**

**To Reproduce**
1. update 0.5.4.1
2. Hwinfo＆Xtream tuner app run
3. app close

**Expected behavior**
QM action　app close

**Screenshots**
Cannot shoot

**Version affected (please complete the following information):**
 - Version number [0.5.4.1]
 - RC 

**NVIDIA driver version**
3090 & 3060Ti

**Hardware**
Describe GPUs used and how are GPUs connected to the motherboard.
CPU?
PCIe slot

**Logs**
Activate logging first, then record event - bug - then export logs into a .zip file and attach it when submitting this issue.
I can't leave a log

**Additional context**

**Comment by iDraKzy** [comment]
Also happens with other apps, happens to me with HWInfo64 and Nvidia Gefore Experience

**Comment by hwatt2** [comment]
While Nicehash Quickminer 0.5.4.1 RC is running, I can confirm this also happens to me with:
- HWMonitor (different versions)
- HWInfo
- CPUz
- All Steam games

Makes sense to think this is an issue with Quickminer 0.5.4.1 RC, as this was not happening with the stable version.

HWMonitor, HWInfo, CPUz and games won't launch even if Nicehash's "Game Mode" is activated, only if Nicehash is **closed** (using "Exit" on tray) and stopping mining all together for all GPUs. For sure, if games won't work in "Game Mode", this can't be an intended behavior.

This issue used to close GeForce Experience as well. However, after a recent Nvidia update, GeForce Experience is working while Nicehash Quickminer is running.

This bug seems to be the same as:
"[BUG] NHQM_v0.5.4.0_RC closes HWiNFO and FanControl #813
(https://github.com/nicehash/NiceHashQuickMiner/issues/813)

Unfortunately, the solution pointed out at #813 was removed from Reddit by Nicehash moderators (why??), as can be seen here:
https://www.reddit.com/r/NiceHash/comments/ukxoyh/fix_for_qmexcavator_054_rc_closing_hwmonitor_nvcp/

Some people could make HWInfo 32bit version work, and I can confirm that this issue is only happening with 64bit version of this program, because I just tested HWInfo 32bit portable and it works. However, even if a temp monitor solution that works can be found, this won't solve the "Game Mode" situation.

If Quickminer is closed, all above mentioned software will work normally. Once Quickminer is opened, the exact second anti-LHR is activated, all "incompatible" software will close immediately.

As I have a Gen1 RTX3060, I was using leaked Nvidia Driver 470.05 with no riser and display connected to prevent LHR. This didn't trigger anti-LHR detection within Quickminer before because the board "didn't know" it had LHR while using this configuration. But this driver was becoming to be too ancient for new games. 100% LHR unlock came at the right time, but I'm thinking about rolling back driver and NHQM versions to make this issue go away.
